globalsearchv2/global_search_v2.go

```diff
@@ -34,22 +34,19 @@ import (
 	common "github.com/IBM/platform-services-go-sdk/common"
 )
 
-// GlobalSearchV2 : Search for resources with the global and shared resource properties repository integrated in the IBM
-// Cloud platform. The search repository stores and searches cloud resources attributes, which categorize or classify
-// resources. A resource is a physical or logical component that can be created or reserved for an application or
-// service instance and is owned by resource providers, such as Cloud Foundry, IBM Kubernetes Service, or resource
+// GlobalSearchV2 : Search for resources with the global and shared resource properties repository that is integrated in
+// the IBM Cloud platform. The search repository stores and searches cloud resources attributes, which categorize or
+// classify resources. A resource is a physical or logical component that can be created or reserved for an application
+// or service instance. They are owned by resource providers, such as Cloud Foundry, IBM Kubernetes Service, or resource
 // controller in IBM Cloud. Resources are uniquely identified by a Cloud Resource Name (CRN)  or by an IMS ID. The
 // properties of a resource include tags and system properties. Both properties are defined in an IBM Cloud billing
 // account, and span across many regions.
 //
-// Version: 2.0.1
+// API Version: 2.0.1
 type GlobalSearchV2 struct {
 	Service *core.BaseService
 }
 
-// DefaultServiceURL is the default URL to make service requests to.
```

Comment:
This default service URL was removed because the "servers" field was removed from the API definition. The "servers" field defines the default service endpoint to be used by users of the service. Why was that removed?

Reply:
<img width="607" alt="image" src="https://user-images.githubusercontent.com/18260697/209148268-bcf53694-63d9-47c0-bf7d-ce37f0cdb889.png">

Reply:
At this question can replay Michele Crudele.

Reply:
@padamstx I believe the team removed the `servers` from the definitions because we use the swaggers to point to our internal test environments, which we cannot reference publicly. Is that a problem not setting the `servers`?

Reply:
Yes, in general one should be using the production version of the API definition when generating code for the SDKs.  Is it a problem to define the production endpoint in the servers field within all three branches (master, test, production)?

Reply:
done

Reply:
done